### PR TITLE
Correction of the article Become a software developer

### PR DESCRIPTION
**Pull request by SHABA-Rex**
Hi everyone,
I hope this time I didn't make any vocabulary mistakes, I checked it 3 times anyway. :)

### CODE REVIEW

File: content/post/articles/Become-A-Software-Developer.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+Bootcamps usually take 12 weeks (full time), it can also be up to 24 weeks (part time).
+
+##### #Pros :
+
+###### Cheaper and Quicker Than a College Degree :
+ Save money and get it done faster? One of the major draws for attending coding bootcamp is its cost effectiveness and the amount of time it takes to graduate.
+ Besides being good for your wallet, bootcamp allows you to get all your coursework done in about 12 weeks for full-time students or 24 weeks for part-time, rather than four years.
+
+###### Practical (Hands-On) Curriculum :
+ Technology is constantly changing—and so must the curriculum. Coding bootcamps offer industry-inspired curriculum that’s constantly updated and applicable to real jobs in the tech world.
```
Comment (RMPR) [inline content/post/articles/Become-A-Software-Developer.md:59]: > Technology is constantly changing—and so must the curriculum

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^

Why the double hyphen here?

File: content/post/articles/Become-A-Software-Developer.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+##### #Cons :
+
+###### Sometimes a degree is the most important :
+Some employers still tend to place greater value on a degree.Either these employers haven’t caught up to the changing times or they see degree holders as more well-rounded and better prepared.
```
Comment (RMPR) [inline content/post/articles/Become-A-Software-Developer.md:64]:
```suggestion
Some employers still tend to place greater value on a degree. Either these employers haven’t caught up to the changing times or they see degree holders as more well-rounded and better prepared.
```

**Review by RMPR — CHANGES_REQUESTED** [review]
Good job overall, still a few things to fix before I can give my approval. Also, you may want [to squash your commits](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits) before this PR is ready to be merged. I noticed you don't use the [bootcamp picture](https://github.com/osscameroon/osscameroon-blog/blob/9609fc54778e03490efe1c79ab18edd5f271f94c/static/Become-A-Software-Developer/Localhost%20academy.png) anymore, consider removing that from the PR as well.

File: content/post/articles/Become-A-Software-Developer.md (hunk 1 above)
Comment (SHABA-Rex) [reply content/post/articles/Become-A-Software-Developer.md:59]: a mistake, i wrote too fast :).

**Review by elhmn — APPROVED** [review]
Great article, I think people will like it.
I removed the spaces between the title and the column, so we will merge this tomorrow morning.

**Review by RMPR — APPROVED** [review]

**Pull request merged**